pkg/logger/logger.go

```diff
@@ -2,34 +2,186 @@ package logger
 
 import (
 	"fmt"
+	"runtime"
+	"strconv"
+	"strings"
+	"time"
 
-	"github.com/fatih/color"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	"github.com/cloudogu/cesapp-lib/core"
+	"github.com/cloudogu/kubectl-ces-plugin/cmd/plugin/cli/util"
+)
+
+// LogLevelKey identifies
+const LogLevelKey = "log-level"
+
+// CLI log level strings
+const (
+	LogLevelError LogLevel = "error"
+	LogLevelWarn  LogLevel = "warn"
+	LogLevelInfo  LogLevel = "info"
+	LogLevelDebug LogLevel = "debug"
+)
+
+// logrus log levels
+const (
+	errorLevel int = iota
+	warningLevel
+	infoLevel
+	debugLevel
 )
 
-type Logger struct {
+var (
+	loggerInstance *NamedLogger
+	formatter      = createFormatter()
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>var `formatter` is unused